tests/test.utils.jsx

```diff
@@ -22,3 +22,39 @@ describe('utils.parseQuery', function() {
   });
 
 });
+
+
+describe('utils.isDisabled', function() {
+
+  it('should be disabled if disabled attr present', function() {
+    var lnk = document.createElement('a');
+    var disabledAttr = document.createAttribute('disabled');
```

Comment:
since this is a jsx file I'd assume you could also do `var lnk = <a disabled />;`